### DISCUSSION TITLE
bowtie2

### DISCUSSION THREAD

**Discussion by paupaiz** [sandbox-bio/sandbox.bio#16] [Q&A] [answered]
Hi Dr. Aboukhalil, Thank you for building sandbox.bio! I am trying the bowtie2 tutorial which worked around a week ago but now I don't have a path set in $REF and the aligner doesn't work: 
<img width="443" alt="Screen Shot 2022-01-28 at 7 30 00 PM" src="https://user-images.githubusercontent.com/59720098/151642040-577429a3-7c7a-4e3d-9859-0646f5d69ff1.png">
 Not sure if it is related but I created an account and signed in (that's the only difference from last time I tried).

**Comment by robertaboukhalil** [comment, marked as answer]
Hi @paupaiz, thanks for reaching out! 

Have you tried refreshing the page and seeing if `echo $REF` works?

If not, can you try typing the command:
``` bash
REF=/bowtie2/example/index/lambda_virus; REF_FASTA=/bowtie2/example/reference/lambda_virus.fa
```
and seeing if it works after that?

**Reply by paupaiz** [reply]
It worked after typing that command!
